rpmbuild failing since commit b7e6ad1abe09a8e86217df4aecbe4a5befb23d37

Hi, I have a build pipeline that checks out the master branch and runs the following commands:

1 - autogen.sh 
2 - configure
3 - make dist
4 - rpmbuild -tb *.gz --define "_topdir `pwd`"

I traced the failure back to this [commit](https://github.com/firehol/netdata/commit/b7e6ad1abe09a8e86217df4aecbe4a5befb23d37).  When the error message was:

```
make[2]: *** No rule to make target `python_modules/urllib3/contrib/_securetransport/__init__.py', needed by `all-am'.  Stop.
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
error: Bad exit status from /var/tmp/rpm-tmp.0A1KEC (%build)
    bogus date in %changelog: Mon Jul 16 2017 Costa Tsaousis <costa@tsaousis.gr> - 1.7.0-1
    Bad exit status from /var/tmp/rpm-tmp.0A1KEC (%build)
```



The latest [commit](https://github.com/firehol/netdata/commit/d75508b1c95a6344411e8aa0c513c58ab769f52e) is erroring with the following error:

```
config.status: error: cannot find input file: `diagrams/Makefile.in'
error: Bad exit status from /var/tmp/rpm-tmp.As4IGJ (%build)
    bogus date in %changelog: Mon Jul 16 2017 Costa Tsaousis <costa@tsaousis.gr> - 1.7.0-1
    Bad exit status from /var/tmp/rpm-tmp.As4IGJ (%build)
```

I will attempt to look into this further next week I just wanted to raise it incase anybody knows what the issue is, or can suggest a better way of building.

--- a/Makefile.am
+++ b/Makefile.am
@@ -48,6 +48,8 @@ EXTRA_DIST = \
 SUBDIRS = \
 	charts.d \
 	conf.d \
+	diagrams \
+	makeself \
 	node.d \
 	plugins.d \
 	python.d \

--- a/conf.d/Makefile.am
+++ b/conf.d/Makefile.am
@@ -21,6 +21,7 @@ dist_nodeconfig_DATA = \
     node.d/named.conf.md \
     node.d/sma_webbox.conf.md \
     node.d/snmp.conf.md \
+    node.d/stiebeleltron.conf.md \
     $(NULL)
 
 pythonconfigdir=$(configdir)/python.d

